tests/migration/migration.go

```diff
@@ -1017,7 +1017,36 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 				Expect(err.Error()).To(ContainSubstring("DisksNotLiveMigratable"))
 			})
 			It("[test_id:1479][storage-req] should migrate a vmi with a shared block disk", decorators.StorageReq, func() {
-				vmi, _ := tests.NewRandomVirtualMachineInstanceWithBlockDisk(cd.DataVolumeImportUrlForContainerDisk(cd.ContainerDiskAlpine), testsuite.GetTestNamespace(nil), k8sv1.ReadWriteMany)
+				if !libstorage.HasCDI() {
+					Skip("Skip DataVolume tests when CDI is not present")
```

Comment:
Thanks for your PR. We should work hard to eliminate the catch-all tests/utils file.

These `Skip`s were here before your PR, but now they are more spread out. `Skip` should not be used in tests, because they may cause us to ignore a test silently and unintentionally. Besides, I bet (but not sure) that the `[storage-req]` label means that these tests are executed only when storage requirements (CDI, block storage) are available. Please avoid them.

Reply:
makes sense , i will change that